src/main/java/mate/academy/services/Book.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.services;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book() {
+
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public BigDecimal getPrice() {
+        return price;
+    }
+
+    public void setPrice(int price) {
+        this.price = BigDecimal.valueOf(price);
```

Comment:
The `setPrice` method takes an `int` as an argument, but the `price` field is a `BigDecimal`. Consider overloading the `setPrice` method to accept a `BigDecimal` directly, or ensure that the conversion from `int` to `BigDecimal` is intentional and documented.